index.js

```diff
@@ -2173,10 +2178,10 @@
   }
   S.maybe_ = def('maybe_', {}, [$.Thunk(b), Fn(a, b), $Maybe(a), b], maybe_);
 
-  //# justs :: Array (Maybe a) -> Array a
+  //# justs :: (Filterable f, Functor f) => f (Maybe a) -> f a
   //.
-  //. Takes an array of Maybes and returns an array containing each Just's
-  //. value. Equivalent to Haskell's `catMaybes` function.
+  //. Discards each of the structure's elements which is Nothing, and unwraps
```

Comment:
It was recently discussed that we could omit mentions of *the structure* from these descriptions, eg: *Discards each element which is Nothing...*; Will we do that in a separate PR?

Reply:
I've updated the descriptions. Please have another look. :eyes: